.pipeline/templates/code-sign.yml

```diff
@@ -0,0 +1,73 @@
+steps:
+  - task: EsrpCodeSigning@2
+    displayName: "Code Signing for windows"
+    condition: eq(variables['platform'], 'windows')
+    inputs:
+      ConnectedServiceName: "Database System ESRP Connector"
+      FolderPath: "$(Build.SourcesDirectory)/dist/pgsqltoolsservice"
+      Pattern: |
+        *.exe
+        *.dll
```

Comment:
I'm assuming you referred to the codesign page before making this change?

https://microsoft.sharepoint.com/teams/prss/esrp/info/ESRP%20Onboarding%20Wiki/Selecting%20CodeSign%20Certificates.aspx

Reply:
oh nvm I see it's still pgsqltoolsservice which is cearly just placeholder for codesign